ontology/vg.ttl

```diff
@@ -113,54 +114,184 @@
   rdfs:label "reverseOfNode"^^xsd:string ;
   rdfs:range :Node ;
 .
+:Bin
+  rdf:type owl:Class ;
+  rdfs:comment "A bin in the output of odgi bin, representing a sequence section."^^xsd:string ;
+  rdfs:label "Bin"^^xsd:string ;
+  rdfs:subClassOf owl:Thing ;
+.
+:Link
+  rdf:type owl:Class ;
+  rdfs:comment "A link marks a graph traversal along a nonlinear connection."^^xsd:string ;
+  rdfs:label "Link"^^xsd:string ;
+  rdfs:subClassOf owl:Thing ;
+.
+:binRank
+  rdf:type owl:ObjectProperty ;
+  rdfs:comment "The rank records that step place along its pangenome sequence."^^xsd:string ;
+  rdfs:domain :Bin ;
+  rdfs:label "binRank"^^xsd:string ;
+  rdfs:range :Bin ;
+.
+:binEdge
+  rdf:type owl:DatatypeProperty ;
+  rdfs:comment "The super property that says two bins are linked on the PangenomeSequence."^^xsd:string ;
+  rdfs:domain :Bin ;
+  rdfs:label "binEdge"^^xsd:string ;
+  rdfs:range xsd:positiveInteger ;
+.
+:arrival
+  rdf:type owl:ObjectProperty ;
+  rdfs:comment "An end bin of nonlinear link. Incoming edge of the bin."^^xsd:string ;
+  owl:inverseOf :departure;
+  rdfs:domain :Link ;
+  rdfs:label "arrival"^^xsd:string ;
+  rdfs:range :Bin ;
+.
+:departure
+  rdf:type owl:ObjectProperty ;
+  rdfs:comment "A start bin of nonlinear link. Outgoing edge of the bin."^^xsd:string ;
+  owl:inverseOf :arrival;
+  rdfs:domain :Link ;
+  rdfs:label "departure"^^xsd:string ;
+  rdfs:range :Bin ;
+.
+:paths
+  rdf:type owl:ObjectProperty ;
+  rdfs:comment "This means that the path participates the non-linear links between bins."^^xsd:string ;
```

Comment:
This sentence is somehow confusing me. What is it you want to say?

Reply:
Link is between two bins, shown as "Arrow" in Schematize. Arrow is shown iff the two bins are not adjacent (I denoted as  non-lenear). Link is composed of multiple paths. So, Link has one or more paths.

Reply:
As you said, this sentence is unclear. Do you have any recommendations?

Reply:
`A list of paths that follow the same non-linear link between two bins`.

Would that make sense?

Reply:
That's it! Thanks!